Client/Frontend/Fakespot/Client/ProductAnalysisData.swift

```diff
@@ -0,0 +1,29 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+
+struct ProductAnalysisData: Codable {
+    let productId: String
+    let grade: String
+    let adjustedRating: Double
+    let needsAnalysis: Bool
+    let analysisUrl: URL
+    let highlights: Highlights
+
+    private enum CodingKeys: String, CodingKey {
+        case productId = "product_id"
+        case grade
+        case adjustedRating = "adjusted_rating"
+        case needsAnalysis = "needs_analysis"
+        case analysisUrl = "analysis_url"
+        case highlights
+    }
+}
+
+struct Highlights: Codable {
+    let price: [String]
+    let quality: [String]
+    let competitiveness: [String]
```

Comment:
`shipping` and `packaging/appearance` are missing here.

Reply:
Is there a json response I can take a look at that contain them? I haven't seen them around.